api/src/main/java/marquez/db/OpenLineageDao.java

```diff
@@ -93,6 +99,14 @@ void createLineageEvent(
       PGobject event,
       String producer);
 
+  @SqlUpdate(
+      "INSERT INTO lineage_events ("
+          + "event_time, "
```

Comment:
Can we default the event type to `DATASET` for `DatasetEvent`s? And/or add a `schema_url` column? When `JobEvent`s support is added, it will be difficult to  distinguish between dataset and job events.

Reply:
Yes, you're right and I noticed this when working on the `JobEvent` and fixed this in:
https://github.com/MarquezProject/marquez/pull/2661/commits/84f46702a05f62ae1c3399ad51d876113006f6e4

It adds `ol_spec_type` column in `lineage_events` (didn't want to use `event_type` which we already use in different context)